security/src/main/java/uk/ac/ox/ndph/mts/security/authentication/SecurityContextUtil.java

```diff
@@ -1,4 +1,4 @@
-package uk.ac.ox.ndph.mts.security.authorisation;
```

Comment:
The SecurityContext is used to store the details of the _currently authenticated user_. It is when we already passed the authentication. So it should be under the authorisation.

Reply:
at the moment it just deals with the authentication?  

getUserId() {
        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
        UserPrincipal userPrincipal = ((UserPrincipal) authentication.getPrincipal());

??
So it should be in authentication?

Authorization and other classes may use it - but its concerns are authentication?